app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -482,7 +482,7 @@ def build_toolbar_hide_button(id)
     return true if %w(container_replicator_edit container_replicator_delete container_replicator_new).include?(id) &&
                    (@record.kind_of?(ContainerReplicator) || @record.nil?)
 
-    return true if %w(container_image_edit container_image_delete container_image_new).include?(id) &&
+    return false if %w(container_image_edit container_image_delete container_image_new).include?(id) &&
```

Comment:
@oschreib we should remove this check if buttons have been deleted from toolbar yaml.